docs/root/intro/version_history.rst

```diff
@@ -34,6 +34,7 @@ Version history
   is set, Envoy will now add or update the grpc-timeout header to reflect Envoy's expected timeout.
 * router: per try timeouts now starts when an upstream stream is ready instead of when the request has
   been fully decoded by Envoy.
+* router: added support for not retrying :ref:`rate limited requests<config_http_filters_router_x-envoy-ratelimited>`.
```

Comment:
Can you add a specific note here that that the rate limit filter sets this header so that if a user was getting retries of rate limited requests before, they will not now. I think this is an appropriate safety change so we don't have to have it be config driven, but we should call it out.